src/sentry/integrations/pipeline.py

```diff
@@ -50,7 +50,11 @@ class IntegrationPipeline(Pipeline):
     provider_manager = default_manager
 
     def finish_pipeline(self):
-        data = self.provider.build_integration(self.state.data)
+        try:
+            data = self.provider.build_integration(self.state.data)
```

Comment:
@EvanPurkhiser the `def error(self, message)` method is on the `Pipeline` not the `PipelineProvider` class. So I did this as it was the smallest change I could think of. Let me know if you believe it should be something else.

Reply:
Can you use `IntegrationError` from the `integrations.exceptions` module?